### ISSUE TITLE
Divider A11y Pass

### ISSUE THREAD

**Issue by 999-Dev** [OfficeDev/office-ui-fabric-react#10570]
Tracking all of the issues brought up with the 09/23 accessibility pass on Divider control
Details on individual bugs can be found on [SharePoint](https://microsoft-my.sharepoint.com/:x:/p/v-amaver/EQ_pzVKBsKVGkITnS4kP9IYByGqqUpgSdPPan3UG_L0TCA?e=YhEAUt) 

Bug 1:[Accessibility][Reflow : Divider]: Text are not visible after resizing the browser to 400%.

**Comment by dzearing** [comment]
After reviewing this, it doesn't seem like a bug worth fixing. It seems like at 400% with a small width, the divider code example, which doesn't even seem to be accessible on the doc site, has a small wrapping but that ends up overlaying the text. The text is visible, and looks fine a most widths at reasonable zoom sizes.
